tests/cypress/tsconfig.json

```
@@ -4,7 +4,7 @@
     "jsx": "react",
     "baseUrl": "./",
     "paths": {
-      "@/*": ["*"],
+      "@/*": ["*", "../../apps/main/src/*"],
```

Please don't make the imports ambiguous

```suggestion
      "@/*": ["*"],
      "@main/*": ["../../apps/main/src/*"],
```

---

This doesn't work, because the app code imported from main doesn't use @main itself, so if there's further imports it can't find them.

1. You import @main/foo.ts
2. But foo imports @lend/bla

Now step 2 fails because it can't find @lend (as you just reconfigured it to @main/lend)